src/ISC.xml

```diff
@@ -6,7 +6,7 @@
          <crossRef>http://www.opensource.org/licenses/ISC</crossRef>
       </crossRefs>
       <titleText>
-         <p>ISC License:</p>
+         <p><alt match="(The )?ISC License( \(ISC[L]?\))?:?"></alt></p>
```

Comment:
This is back to having no canonical title in the license body (which seems reasonable to me, but has had a fair bit of discussion in this PR, I don't know if there is a consensus yet).  If the intention *is* to have no canonical title in the license body, I think we want to drop the `<p>` tags too.  No need to render an empty `<p></p>`.

If the intention is to have a canonical title in the license body, then I think you want:

```html
<p><alt match="(The )?ISC License( \(ISC[L]?\))?:?">ISC License</alt></p>
```

or whatever you decide to use for the canonical title inside.  I'm also fine with `<alt …><titleText><p>…</p></titleText></alt>` if folks prefer that order.

Reply:
I'd prefer the title be in the canonical version, so I'll go ahead and add it back. @goneall can either approve or ask it to be removed.